src/looprestoration_tmpl_8.rs

```diff
@@ -134,144 +134,12 @@ use crate::src::looprestoration::padding;
 use crate::src::looprestoration::wiener_erased;
 use crate::src::looprestoration::Dav1dLoopRestorationDSPContext;
 use crate::src::looprestoration::LooprestorationParams;
+use crate::src::looprestoration::{boxsum3, boxsum5};
```

Comment:
Can you split these into one line each?  I've been doing that since there's a ton of imports usually and I find it easier to read that way when there are so many, plus the diffs are way cleaner.

Reply:
Same for 16bpc, too.

Reply:
Done!